Community (#163)
codewars/docs: content/community/teams.md

---
prev: /community/
next: /community/rules/
---

# Teams and Organization

_This page is a stub._

## Administrators

Administrators are people who created, maintain, and are responsible for Codewars overall. They handle the most critical issues, like:
- severe cases of violations of rules,
- user accounts,
- bugs,
- ...?

Cuttently, users with status of Administrator are:

- [kazk](link to Codawars profile?)

> - ... others?

Reviewer: jhoffner, alchemy, OverZealous. I think that's it or did I forget someone?

Author: Yeah I am just not sure if all of them are/want to be involved.

If you have any ideas on the topic, feel free to drop them in #98. Once discussion settles down, I will put ideas into this doc, un-draft the PR, and start sending review requests :)

Thanks!

---
prev: /community/
next: /community/rules/
---

> # Teams and Organization

Reviewer 2: After reviewing, I think it's better to:

- rename this file to `content/community/moderation.md`
- change the title to `# Moderation`
- summarize how we moderate, briefly introduce roles (`admin` and `mod`)

Then

- further explain admins
  - maybe include some history and mention the Qualified team (can be done later)
- further explain mods
- explain how to contact us

Currently, users with status of Moderator are:

- [Blind4Basics](https://www.codewars.com/users/Blind4Basics)
- [Chrono79](https://www.codewars.com/users/Chrono79)
- [donaldsebleung](https://www.codewars.com/users/donaldsebleung)
- [g964](https://www.codewars.com/users/g964)
- [hobovsky](https://www.codewars.com/users/hobovsky)
- [monadius](https://www.codewars.com/users/monadius)
- [steffan153](https://www.codewars.com/users/steffan153)
- [Unnamed](https://www.codewars.com/users/Unnamed)

They can be recognized by symbol of a shield (🛡️) displayed by their user name.

## How to get in touch

All kinds of issues are to be reported to [Codewars Github board](https://github.com/codewars/codewars.com/issues), using appropriate issue template if available. There are already issues dedicated to handling [spammers](https://github.com/codewars/codewars.com/issues/1493) and [cheaters](https://github.com/codewars/codewars.com/issues/1378).

> Issues which should not be exposed public or should be brought directly to attention of Administrators should be reported via email to <info@codewars.com>.

Reviewer 3: ```suggestion
Issues which should not be exposed publicly or should be brought directly to the attention of Administrators should be reported via email to <info@codewars.com>.
```

> Many members of moderation team (but not all of them) can be reached via [Gitter](https://gitter.im/Codewars/codewars.com). Message them directly or post on one of the public channels if you need their assistance.

Reviewer 3: ```suggestion
Many members of the moderation team (but not all of them) can be reached via [Gitter](https://gitter.im/Codewars/codewars.com). Message them directly or post on one of the public channels if you need their assistance.
```

# Teams and Organization

## Administrators

Administrators are people who created, maintain, and are responsible for Codewars overall. They handle the most critical issues, like:
- Development and maintenance of Codewars platform,
- Fixing platform bugs related to Codewars functions, user interface, code runner,
- Managing user accounts,
- Handling severe cases of violations of rules.

Cuttently, active users with status of Administrator are:

- [kazk](https://www.codewars.com/users/kazk)

They can be recognized by symbol of a crown (👑) displayed by their user name.

## Moderators

Moderators are small team of users selected by Administrators to support them in handling some tasks related to community and Codewars content. Moderators have tools and privileges which help with maintenance of content and with handling of some community issues. Moderators have access to tools which allow them to:
- Moderate discourses,
- Manage submitted solutions: re-test them, flag as cheats, etc.
- Identify cheaters, plagiarists, and users who do not play the game fair,
- Handle abusers and revoke their privileges,

> - Maintain documentation.

Reviewer 2: > - Maintain documentation.

This might need to be clarified. Anyone can contribute to the docs by opening PRs. Moderators are responsible for merging/rejecting them.

Moderators also serve as intermediaries (word?) between Codewars users and Administrators, or vice versa. They can pass anouncements from maintainers, or support users in communication when some issue needs to be brought to attention of Administrators.

But still, Moderators are just regular users, and as such, they're subject to all Codewars rules. In particular, they **cannot**:
- Fix problems with kata they have no privileges to coauthor,
- Approve translations which cannot be normally approved,

> - Modify kata in a way which is not directly supported by Codewars: change its ranking, move it back to beta, retire or delete it.

Reviewer 2: Do we need this section? Seems redundant to me because there's already a section about what mods can do.

Also,

> - Fix problems with kata they have no privileges to coauthor,

I'll allow this in the future.

> - Approve translations which cannot be normally approved,

I might allow this in the future if it makes sense. Maybe require more than one user to approve.

> - Modify kata in a way which is not directly supported by Codewars: change its ranking, move it back to beta, retire or delete it.

I might allow moving back to beta if it's within few hours and if multiple mods agree. This will allow reranking for new kata.

I might also allow manually retiring beta kata that's been stuck for a while and when it's unlikely to be ever approved.

Author: I included it to limit situations when mods get flooded with requests for handling situations they cannot handle.

It's kinda preemptive, so I will remove it now and add back if it turns out necessary ;)

Currently, users with status of Moderator are:

- [Blind4Basics](https://www.codewars.com/users/Blind4Basics)
- [Chrono79](https://www.codewars.com/users/Chrono79)
- [donaldsebleung](https://www.codewars.com/users/donaldsebleung)
- [g964](https://www.codewars.com/users/g964)
- [hobovsky](https://www.codewars.com/users/hobovsky)
- [monadius](https://www.codewars.com/users/monadius)
- [steffan153](https://www.codewars.com/users/steffan153)
- [Unnamed](https://www.codewars.com/users/Unnamed)

They can be recognized by symbol of a shield (🛡️) displayed by their user name.

## How to get in touch

All kinds of issues are to be reported to [Codewars Github board](https://github.com/codewars/codewars.com/issues), using appropriate issue template if available. There are already issues dedicated to handling [spammers](https://github.com/codewars/codewars.com/issues/1493) and [cheaters](https://github.com/codewars/codewars.com/issues/1378).

> Issues which should not be exposed public or should be brought directly to attention of Administrators should be reported via email to <info@codewars.com>.

Reviewer 2: ```suggestion
Issues which should not be exposed public or should be brought directly to attention of Administrators should be reported via email to <info@codewars.com>. For security issues, please read our [security policy](https://github.com/codewars/codewars.com/security/policy).
```
